Sources/PaymentButtons/PaymentButton+ImageAsset.swift

```diff
@@ -34,14 +34,8 @@ extension PaymentButton {
         }
 
         switch color {
-        case .gold, .white, .silver:
```

Comment:
From what I see, the switch seems unnecessary. I think we can remove it. What do you think?

Reply:
Good point thanks!